src/mainboard/amd/romecrb/src/ioapic.rs

```diff
@@ -0,0 +1,61 @@
+use core::ptr;
+
+fn ioapic_read(ioapic_base: u32, register: u32) -> u32 {
```

Comment:
TODO: separate the basic generic ioapic_read and _write (can't it be a driver?) from the specific details for this board